include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -318,10 +316,9 @@ bool print_transaction_table(Transaction & transaction) {
             scols_line_set_data(ln_replaced, COL_ARCH, replaced.get_arch().c_str());
             scols_line_set_data(ln_replaced, COL_EVR, replaced.get_evr().c_str());
             scols_line_set_data(ln_replaced, COL_REPO, replaced.get_repo_id().c_str());
-            uint64_t size = replaced.get_install_size();
-            scols_line_set_data(
-                ln_replaced, COL_SIZE, libdnf::cli::utils::units::format_size(static_cast<int64_t>(size)).c_str());
 
+            auto replaced_size = static_cast<int64_t>(pkg.get_install_size());
```

Comment:
You are taking wrong package object.
Please change `replaced.get_install_size()` to `pkg.get_install_size()`.

Reply:
What an oversight, thanks! Fixed.